rocksdb_admin/admin_handler.cpp

```diff
@@ -789,7 +791,7 @@ void AdminHandler::async_tm_addS3SstFilesToDB(
   std::shared_ptr<common::S3Util> local_s3_util;
```

Comment:
can we just do the following here.

`if (FLAGS_s3_download_limit_mb > 0) { request-> s3_download_limit_mb = FLAGS_s3_download_limit_mb; }`

Reply:
I meant to put it immediately after line 791. Then we don't need other changes any more.

Reply:
good point, updated

Reply:
Do a max check on the FLAG value before setting it to the s3_request ?

Reply:
it depends on do we want to follow the rule "gflag as first priority" or not.

Reply:
Regardless of whether we use GFLAGS as the first priority or not, it might be better to add a check against a max download limit size (2G / 5G) to make sure that if we accidentally set the value too high in GFLAG we will never cross a max limit

Reply:
Yes, may have another pull request for this check, besides glflag, our request level also needs this kind of check